docs/source-app/examples/github_repo_runner/app.py

```diff
@@ -24,7 +24,7 @@ def __init__(
         script_args: List[str],
         requirements: List[str],
         cloud_compute: Optional[CloudCompute] = None,
-        **kwargs: Any,
+        **kwargs,
```

Comment:
why remove typing?

Reply:
@tchaton ?

Reply:
I don't think it really matters, but we can keep it.

Reply:
Done !